tcms/testruns/static/testruns/js/get.js

```diff
@@ -157,6 +170,12 @@ $(document).ready(() => {
   })
 
   $('.js-toolbar-filter-options li').click(function (ev) {
+    if ($(ev.currentTarget).data('filter-type') === 'assignee__username') {
+      $('#assigned-to-me-filter').css('display', 'block')
+    } else {
+      $('#assigned-to-me-filter').css('display', 'none')
+    }
```

Comment:
IMO the logic here is backwards. You don't want to show & hide extra widgets depending on if the user selects filtering by assignee. You want to have an on/off button that would issue a filter query directly with the appropriate parameters.

This is also what the issue asks for and how many other apps work.

Reply:
so basically have another checkbox next to the filters that says "assigned to me" and triggers this logic?

Reply:
I reworked the UI. I did it in a separate commit if I need to go back to the old code. once we decide it's ready to merge I can squash the changes.